extensions/bouncer.md

```diff
@@ -0,0 +1,236 @@
+---
+title: IRCv3 bouncer extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Darren Whitlen"
+    period: "2018-2021"
+    email: "darren@kiwiirc.com"
+---
+
+## BOUNCER
+
+This feature is intended to provide an API for clients to build a UI around bouncer IRC servers, to manage the users IRC networks and buffers in a user friendly way and to keep multiple clients and devices synced together.
+
+It is based around network and buffer entities which both have key/value tags, such as `joined=true` for a buffer, with commands to manage these. A list of common tags can be found under the tags section of this document.
+
+From a single bouncer connection, a complete implementation will allow users to:
+* Add, remove, modify and search networks under their bouncer account.
+* List and search channels and queries.
+
+
+## Notes
+References in this document:
+- A `network` is an IRC network that a bouncer connects to
+- A `buffer` is either a channel or query
+- A client is a users client or device that connects to the bouncer
+
+A network is identified by a per-user unique ID that MUST not change. A human readable label MUST also be associated with the network that the user can identify it with in a client UI - this is the `network` tag in most commands. The ID and label may be the same as long as the ID does not change. Neither the ID or label may contain a space or `:` characters.
+
+All commands and data for this bouncer feature MUST be associated to a logged-in and authorised user. It is the implementors choice on how the user authentication is handled and is outside the scope of this document.
+
+## Capabilities
+
+This adds a capability called `BOUNCER`. Once negotiated the server may change it's behaviour to better suit bouncer clients, such as not automatically sending `JOIN` commands for already joined channels.
```

Comment:
Caps are lowercase. Additionally, this is missing the "work in progress" section about using the `draft/`-prefixed cap.